src/main/java/org/mule/extension/api/introspection/ExpressionSupport.java

```diff
@@ -28,5 +28,10 @@
     /**
      * Requires expressions. Static values are not supported
      */
-    REQUIRED
+    REQUIRED,
+
+    /**
+     * Expressions are allowed but are going to be treated as literals
```

Comment:
treated as literals -> are not going to be automatically evaluated. The value will be supplied into the OperationExecutor on its literal form